grobid-core/src/main/java/org/grobid/core/data/BiblioItem.java

```diff
@@ -1973,9 +1973,12 @@ public String toBibTeX(String id, GrobidAnalysisConfig config) {
             }
             // fullEditors has to be used instead
 
-            // year
-            if (publication_date != null) {
-                bibtex.add("  year = {" + publication_date + "}");
+            // dates
+            if (normalized_publication_date != null) {
+                String isoDate = normalized_publication_date.toISO();
```

Comment:
There's already an ISO normalization method to be re-used: 
https://github.com/kermitt2/grobid/blob/master/grobid-core/src/main/java/org/grobid/core/document/TEIFormatter.java#L108